scripts/qemu/start_qemu.sh

```diff
@@ -3,28 +3,25 @@
 # Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-set -x
 set -e
 
-config_file=$1
+# The config file name (should be located in ./configs/ sub-dir)
+config_name=$1
 
-python3 scripts/qemu/qemu_config.py $config_file
+# Parse the config file to get topology info and fix escaped single quotes
+parsed_config=$(python3 "$(dirname $0)/parse_config.py" "$(dirname $0)/configs/${config_name}" | sed s/''\''/'/g)
```

Comment:
nit: maybe print out this `parsed_config` ?

Reply:
the whole command, along with the parsed part, is printed, thanks to `set -x`